packages/user-group-service/src/types.d.ts

```diff
@@ -77,3 +78,18 @@ type UserType = {
   updatedBy: string;
   updatedOn: Date;
 }
+
+type Group = {
+  _id: any;
+  name: string;
+  ldapCommonName: string;
```

Comment:
Can you use it as ```cn``` if we can make the field name as same as ldap that will be good